build/distributions/CentOS/7/x86_64/DVD/ks/dvd/auto.cfg

```diff
@@ -4,7 +4,7 @@ text
 keyboard us
 lang en_US
 
-%include /mnt/install/repo/ks/dvd/include/common_ks_base
+%include /mnt/stage2/ks/dvd/include/common_ks_base
```

Comment:
This was changed in EL7 to 'install'. Stage2 may have reared its ugly head again, but it definitely did not work in some versions.